.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Pull Request
```

Comment:
would be good to decide on a title convention like`feature|chore|bugfix(context): title [ticket]` maybe?
e.g. `feature(button): add new theme to the button [HEYUI-999]`

Reply:
Yeah, that's a good point

Reply:
It seems to enforce a PR title we would need a tool like this: https://github.com/marketplace/semantic-pull-requests

Reply:
hmmm got it. i don't it's really _that_ necessary, but something to have in mind then. thanks!!